### ISSUE TITLE
Verify if the user authorized all the required scopes

### ISSUE THREAD

**Issue by ricardobrg** [tiki/strategy_google#77]
When the user returns from the Google login flow, the app needs to verify if the authorization has all the required scopes.

If it lacks any permission, show the UI defined in #75 and let the user retry.

Acceptance criteria
1 - The UI shows when the user doesn't give all permissions.

**Comment by NickkTMD** [comment]
Really, the user should not be allowed to login without authorizing all required scopes.

There are a lot of threads on this issue.

One thing that worked for some people is setting the scopes we need explicitly in
https://console.cloud.google.com/apis/credentials/consent/edit?project=%7BYourProjectID%7D
though that doesn't always work and may have already been done

Another option is switching login flow to first authorize basic scopes, then request more. This can be done natively with
https://pub.dev/packages/google_sign_in, though we've had issues with authorizing gmail scopes with them in the past.

Created new issue #86 for this

**Comment by NickkTMD** [comment]
Detecting scopes if now done:
https://github.com/tiki/strategy_google/commit/4b7b941bd0b44e658b8d3d9477755cf698b79458

So this issue is blocked by #75
